a10_neutron_lbaas/instance_manager.py

```diff
@@ -161,7 +197,20 @@ def _create_instance(self, context, instance):
         server["image"] = image.id
         server["flavor"] = flavor.id
         server["networks"] = networks
-        return self._nova_api.servers.create(**server)
+        created_instance = self._nova_api.servers.create(**server)
+        # Next 6 lines -  Added due to insane API on the other side
+        created_instance.manager.client.last_request_id = None
```

Comment:
We need to test this against master. The files for the api stuff have changed a lot since the ones I looked up in liberty.